Automattic/wp-calypso
# Make universal footer save.js ready

## Proposed Changes

* This makes the Universal Footer save.js ready by removing all dependency on state, hooks, and any React reactivity. 
* It makes the happy-blocks follow the site language, not the user's.

## Testing Instructions
1. Go to this PR 312-gh-Automattic/wpsupport3.
2. set your env to run it (checkout and `npm run dev`).
---
1. cd into `apps/happy-blocks`.
2. run `yarn dev --sync`.
4. Sandbox es.support.wordpress.com.
5. Visit https://es.support.wordpress.com/wp-admin/post.php?post=5466&action=edit.
6. Universal footer should be in Spanish. 
7. Visit https://wordpress.com/es/support/test/, same.
----
#### Regression testing

0. Run Calypso locally.
1. Go to /plugins and themes.
2. Make sure the footer and header work perfectly. Compare against prod.
